AlertaDengue/dados/templatetags/home_components.py

```diff
@@ -7,7 +7,7 @@
 
 @register.inclusion_tag("components/home/collapse.html", takes_context=True)
 def collapse_component(context):
-    context["states_name"] = dbdata.STATE_NAME
+    context["states_name"] = dict(sorted(dbdata.STATE_NAME.items(), key=lambda state: state[1]))
```

Comment:
I think you will have to reference this issue #561 in that same PR.

Reply:
You mean to separate this fix on another PR?

Reply:
Not because it is a small update. But it is not recommended to treat different issues in the same PR.